statics/js/components/alert-form.js

```diff
@@ -0,0 +1,75 @@
+/* jshint multistr: true */
+
+Vue.component('alert-form', {
+
+  mixins: [apiMixin, notificationMixin],
+
+  template: '\
+    <div>\
+      <form @submit.prevent="start" class="alert-form" v-if="visible">\
+        <div class="form-group">\
+          <label for="alert-name">Name</label>\
+          <input id="alert-name" type="text" class="form-control input-sm" v-model="name" />\
+        </div>\
+        <div class="form-group">\
+          <label for="alert-desc">Description</label>\
+          <textarea id="alert-desc" type="text" class="form-control input-sm" rows="2" v-model="desc"></textarea>\
+        </div>\
+        <div class="form-group">\
+          <label for="alert-expr">Expression</label>\
+          <a><i class="fa fa-question help-text" aria-hidden="true" title="Gremlin or JavaScript expression evaluated to trigger the alarm"></i></a>\
+          <textarea id="alert-expr" type="text" class="form-control input-sm" rows="3" v-model="expr"></textarea>\
+        </div>\
+        <div class="form-group">\
+          <label for="alert-trigger">Trigger</label>\
+          <a><i class="fa fa-question help-text" aria-hidden="true" title="Event that triggers the alert evaluation"></i></a>\
+          <input id="alert-trigger" type="text" class="form-control input-sm" v-model="trigger" />\
+        </div>\
+        <div class="form-group">\
+          <label for="alert-action">Action</label>\
```

Comment:
It could be nice to add "help tittle" for this too. Maybe an URL prefix like "IP" for the generator

Reply:
done